manifests/a/AnyAssociation/Anytype/0.35.2/AnyAssociation.Anytype.locale.en-US.yaml

```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: AnyAssociation.Anytype
+PackageVersion: 0.35.2
+PackageLocale: en-US
+Publisher: Any Association
+PublisherUrl: https://github.com/anyproto
+PublisherSupportUrl: https://anytype.io/faq
+PrivacyUrl: https://anytype.io/website_privacy
+Author: Any Association
+PackageName: Anytype
+PackageUrl: https://download.anytype.io/
+License: Proprietary
+Copyright: Copyright © 2023 Any Association
+ShortDescription: The everything app for those who celebrate trust & autonomy.
+Documentations:
+- DocumentLabel: Anytype Docs (English)
+  DocumentUrl: https://doc.anytype.io/anytype-docs/intro/readme
+- DocumentLabel: Anytype 文档（简体中文）
+  DocumentUrl: https://doc.anytype.io/anytype-docs/v/documentation_cn/

```

Comment:
```suggestion
```

Remove online document entrance of Simplified Chinese. Users can switch languages on their own.